packages/web/src/lib/dispatcherPreferences.ts

```
@@ -0,0 +1,323 @@
+      }
+    } catch (error) {
+      if (!disposed) {
+        options.onError?.(error, payload);
+      }
+    } finally {
+      state.inFlight = false;
+      if (disposed) {
+        scopeStates.clear();
+        scopePending.clear();
+        resolveIdle();
+        return;
```

Biome (2.5.3)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)